packages/AlgernonTests.package/ALGTestTypes.class/instance/testExpressionDoNoTranscript.st

```diff
@@ -1,6 +1,5 @@
 testing expressions
 testExpressionDoNoTranscript
```

Comment:
generally I think the newline is not considered to be bad pratice - but as you changed it we can jus leave it as it is :)

Reply:
Yes.